requirements.txt

```diff
@@ -1,3 +1,3 @@
 discid
 mutagen>=1.37
-PyQt5>=5.7.1,<5.11
+PyQt5>=5.7.1
```

Comment:
I would actually like to freeze the build dependencies. This prevents unknown regressions (this happened during the 2.0.1 release with osx). Maybe a requirements-prod-win or something like that? And similar file for osx.